scripts/ci.go

```diff
@@ -0,0 +1,71 @@
+// +build none
+
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"runtime"
+	"strings"
+)
+
+const (
+	GOLANGCI_VERSION = "github.com/golangci/golangci-lint/cmd/golangci-lint@v1.23.1"
+)
+
+func main() {
+
+	log.SetFlags(log.Lshortfile)
+
+	if _, err := os.Stat(filepath.Join("scripts", "ci.go")); os.IsNotExist(err) {
+		log.Fatal("should run build from root dir")
+	}
+	if len(os.Args) < 2 {
+		log.Fatal("cmd required, eg: install")
+	}
+	switch os.Args[1] {
+	case "lint":
+		lint()
+	default:
+		log.Fatal("cmd not found", os.Args[1])
+	}
+}
+
+func lint() {
+
+	verbose := flag.Bool("v", false, "Whether to log verbosely")
```

Comment:
```suggestion
        var verbose bool
	flag.BoolVar(&verbose, "v", false, "whether to log verbosely")
```

Reply:
Doing that still requires the pointer. I just tried.